### DISCUSSION TITLE
mpv.conf 中如何给多层级的复杂选项赋值

### DISCUSSION THREAD

**Discussion by verygoodlee** [hooke007/mpv_PlayKit#411] [mpv 通用]
例如 [user-data](https://mpv.io/manual/master/#command-interface-user-data) [chapter-list](https://mpv.io/manual/master/#command-interface-chapter-list) 这些可写的多层级选项，\
在lua脚本就是table类型，很容易操作，但是在mpv.conf中或者启动参数中不知道怎么填，
试了`user-data/my-script/key=value` `user-data['my-script']['key']=value` `user-data.my-script.key=value` 这几种写法，都是报语法错误，\
是有啥特殊写法吗，还是压根就不支持。

**Comment by hooke007** [comment]
这两就不是Option，只是prop

有前导符号 -- 的才是 选项 。

**Reply by verygoodlee** [reply]
确实，没注意到，以为跟`vf` `af`类似，`vf` `af`他它俩既是属性也是选项

**Reply by hooke007** [reply]
每个option也大概率可以是prop，这个在手册里有写的。
prop list那边列出的prop是一些特例

**Comment by verygoodlee** [comment]
mpv.conf中找到一种取巧方法了，利用`profile-cond`可执行lua脚本的特性，实测可行
```conf
[test]
# 不会检测任何属性变化，只会在启动时执行一次
profile-cond=mp.set_property_native('user-data/my-script/key', 'value')
```
启动参数就确实没办法了

**Reply by hooke007** [reply]
启动肯定不行，，属性都要在初始化之后才有（
